src/Client.php

```diff
@@ -57,6 +57,7 @@ public function store(GotenbergRequestInterface $request, string $destination):
         $response = $this->handleResponse($this->client->sendRequest($this->makeMultipartFormDataRequest($request)));
         $fileStream = $response->getBody();
         $fp = fopen($destination, 'w');
+        assert(is_resource($fp));
         fwrite($fp, $fileStream->getContents());
```

Comment:
@gulien Without `Safe` the calls to `fwrite`, `fclose` etc... now don't check the result isn't `false`, do you think it's worth adding these checks here instead?

Reply:
I think it is worth adding those checks indeed ; fail fast and fail loud 👍

Reply:
Done, I borrowed the `FilesystemException` class from Safe.